UPGRADE.md

```diff
@@ -0,0 +1,71 @@
+### Kubevirt HCO upgrade procedure
+
+##### Summary
+
+In order to tell the OLM to start the upgrade process, one needs to point the relevant operator subscription to the updated CSV (ClusterServiceVersion).
+
+It may be as simple as updating the subscription's **channel**, inside the currently used **catalog**. Or, in other cases, it may require updating the **catalogsource** first, to point to the updated image, one that will contain a new catalog, with a CSV for this new channel.
+
+Once OLM will recognize that the subscription points to a new CSV, it will attempt to peform an update. By default this happens automatically, but this behaviour can be modified for each operator, by modifying the relevant **subscription**, in case one wants to manually approve each update.
```

Comment:
This seems a bit off to me. My understanding was that OLM is constantly searching through the different operator catalogs looking for updates to operators. Once it finds an update, it starts going through the process of updating that operator.